libs/server-sent-events/src/client.cpp

```diff
@@ -264,7 +264,7 @@ class FoxyClient : public Client,
                 return async_backoff(backoff_reason(response.result()));
```

Comment:
When does this get logged and what does it look like if it is?

Reply:
It's what happens when there's a "recoverable" client error and a retry will occur. The log message looks like: 
> "Backing off in (N) seconds due to HTTP status 403 (Forbidden)"

Reply:
Cool. Thank you.